assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-terms-block/test/edit.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import { render, findByRole, queryByText } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Edit } from '../edit';
+const blockSettingsMock = jest.requireMock( '@woocommerce/block-settings' );
+
+jest.mock( '@wordpress/block-editor', () => ( {
+	...jest.requireActual( '@wordpress/block-editor' ),
+	useBlockProps: jest.fn(),
+} ) );
+
+jest.mock( '@woocommerce/block-settings', () => ( {
+	...jest.requireActual( '@woocommerce/block-settings' ),
+	PRIVACY_URL: '/privacy-policy',
+	TERMS_URL: '/terms-and-conditions',
+} ) );
+
+describe( 'Edit', () => {
+	it( 'Renders a checkbox if the checkbox attribute is true', async () => {
+		const { container } = render(
+			<Edit
+				attributes={ {
+					text: 'I agree to the terms and conditions',
+					checkbox: true,
+				} }
+				setAttributes={ () => {} }
```

Comment:
Not a big deal, but [I think we agreed](p1638224334081700/1638213487.071400-slack-C8X6Q7XQU) to use `() => void 0` for empty callback functions as it plays nicely with typescript. In fact the return type should be `void` as [typed here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-terms-block/edit.tsx#L27)